docs/index.rst

```diff
@@ -386,11 +386,19 @@ version and platform itself.
 
 .. image:: versions.png
 
-Settings
-~~~~~~~~
+.. _panel_settings:
+
+Settings Panel
```

Comment:
It is unnecessary to rename because it is already under the heading of "Toolbar Panels". Also it will blow out the left navigation and would be inconsistent with other panel subheadings.

```suggestion
Settings
```